src/Source.ruleset

```diff
@@ -10,7 +10,6 @@
     <Rule Id="CA1016" Action="Warning" />
     <Rule Id="CA1033" Action="Warning" />
     <Rule Id="CA1049" Action="Warning" />
-    <Rule Id="CA1060" Action="Warning" />
```

Comment:
This change disable CA1060 on ML.NET repository, since there are several errors thrown by this rule.

This is an example of what was mentioned here: https://github.com/dotnet/machinelearning/pull/5331#discussion_r462694774 . CA1060 isn't enabled in the security ruleset from FxCop analyzers, but it was previously enabled on the `src/Source.ruleset` file by former members of ML.NET... problem is that since FxCop analyzers weren't installed previously on ML.NET, this rule was never previously enforced... when installing FxCop, it is now enforced and throw several errors... but since it isn't part of the actual security ruleset from FxCop, @sharwell advices to simply disable the rule. This might be the case for other rules as well.